README.np.md

```diff
@@ -0,0 +1,118 @@
+# अपोलो -11
+
+[![NASA][1]][2]
+[![SWH]][SWH_URL]
+[![Comanche]][ComancheMilestone]
+[![Luminary]][LuminaryMilestone]
+
+🎌
+[Bahasa Indonesia][ID],
+[Català][CA],
+[Čeština][CZ],
+[Deutsch][DE],
+[English][EN],
+[Español][ES],
+[Français][FR],
+[Italiano][IT],
+[Kurdi][KU],
+[Nederlands][NL],
+[Polski][PL],
+[Português][PT_BR],
+[Română][RO],
+[Tiếng Việt][VI],
+[Türkçe][TR],
+[Русский][RU],
+[العربية][AR],
+[فارسی][FA],
+**हिंदी**,
+[বাংলা][BD_BN],
+[မြန်မာ][MM],
+[日本語][JA],
+[正體中文][ZH_TW],
+[简体中文][ZH_CN],
+[한국어][KO_KR]
+
+[AR]:README.ar.md
+[BD_BN]:README.bd_bn.md
+[CA]:README.ca.md
+[CZ]:README.cz.md
+[DE]:README.de.md
+[EN]:README.md
+[ES]:README.es.md
+[FA]:README.fa.md
+[FR]:README.fr.md
+[HI_IN]:README.hi_in.md
+[ID]:README.id.md
+[IT]:README.it.md
+[JA]:README.ja.md
+[KO_KR]:README.ko_kr.md
+[KU]:README.ku.md
+[MM]:README.mm.md
+[PL]:README.pl.md
+[PT_BR]:README.pt_br.md
+[RO]:README.ro.md
+[RU]:README.ru.md
+[TR]:README.tr.md
+[VI]:README.vi.md
+[ZH_CN]:README.zh_cn.md
+[ZH_TW]:README.zh_tw.md
+[NL]:README.nl.md
+
+अपोलो ११ मार्गदर्शन कम्प्यूटर (Apollo 11 Guidance computer, AGC) कमाण्ड मोड्यूल   (Comanche055) र चंद्र मोड्युल (Luminary099)  को लागी स्रोत कोड। मान्छेहरू द्वारा अंकित [Virtual AGC][3] र [MIT Museum][4] लक्ष्य मूल अपोलो ११ स्रोत कोडका लागि रेपो हुनु हो। त्यस्तै, PR हरू यस भण्डारमा ट्रान्सक्रिप्शन र मूल स्रोत स्क्यानका बीच पहिचान गरिएको कुनै पनि मुद्दाहरूको लागि स्वागत छ [Luminary 099][5] र [Comanche 055][6] साथै कुनै पनि फाईलहरू मैले छुटेको हुन सक्छु।
+
+## योगदान
+
+ कृपया[CONTRIBUTING.md][7] एक पुल अनुरोध खोल्न अघि पढ्नुहोस्।
+
+## कम्पाइल गर्दै
+
+यदि तपाईं मूल स्रोत कोड कम्पाइल गर्न रूचि राख्नुहुन्छ भने,[Virtual AGC][8] को जाँच गर्नुहोस्।
+
+## एट्रिब्युसन
+
+&nbsp;      | &nbsp;
+:---------- | :-----
+कॉपीराइट      | पब्लिक डोमेन
+Comanche055 | कोलोसस २ ए को लागी स्रोत कोड को अंश, कमाण्ड मोड्युल (सीएम) अपोलो गाइडन्स कम्प्युटर (AGC) अपोलो ११ को लागी <br>`AGC प्रोग्राम Comanche को NASA द्वारा संशोधन ० 055 सम्मिलित`<br>`2021113-051. 10:28 APR. 1, 1969`
+Luminary099 |  ल्यूमिनरी १ए, अपोलो ११ के चंद्र भाग (LM) के लिए निर्देशरण संगड़क यंत्र (AGC) के नियमसंग्रह का हिस्सा। <br>`Luminary 1A को लागि स्रोत कोडको भाग, चंद्र मोड्युलको (LM) अपोलो गाइडन्स कम्प्युटर (AGC) अपोलो ११ को लागि`<br>`2021112-061. 16:27 JUL. 14, 1969`
+Assembler     | yaYUL
+संपर्क     | रोन बुर्की (Ron Burkey) <info@sandroid.org>
+वेबसाइट      | www.ibiblio.org/apollo
+डिजिटलीकरण   | यो स्रोत कोड ट्रान्सक्रिप्ट गरिएको छ वा अन्यथा एमआईटी संग्रहालयबाट हार्डकपीको डिजिटलाइज्ड छविहरूबाट रूपान्तरण गरिएको छ। डिजिटलाइजेशन फिल फेजेल(Paul Fjeld)द्वारा गरिएको थियो, र संग्रहालयको डेबोरा डगलस (Deborah Douglas) द्वारा व्यवस्था गरिएको थियो। दुबैलाई धेरै धन्यवाद।
```

Comment:
It's preferred to not transliterate people names on this repo

Reply:
Got it